github/orgs.go

```diff
@@ -56,6 +56,10 @@ type Organization struct {
 	// MembersCanCreateRepos default value is true and is only used in Organizations.Edit.
 	MembersCanCreateRepos *bool `json:"members_can_create_repositories,omitempty"`
 
+	// MembersAllowedRepositoryCreationType denotes if organization members can create repositories
+	// and the type of repositories they can create.
```

Comment:
Please add to this comment: `Possible values are: "all", "private", or "none".`

Reply:
👍